Production_scripts/Process_extracts/Produce_extract_tests/extract_ae_tests.R

```diff
@@ -0,0 +1,45 @@
+####################################################
+# A&E Extract Tests
+# Original Authors - Catherine Holland
+# Original Date - May 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               A&E processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+library(openxlsx)
+library(slfhelper)
+
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "AE", ext = "rds"))
+# zsav until rds file avaliable
+new_data <- haven::read_sav(get_source_extract_path(year, "AE", ext = "zsav")) %>%
+  rename(postcode = "Postcode")
```

Comment:
I think you can run the rds file and remove this line, i just deleted the rds files in the folder for space but if you need it then this can be ran again. Let me know if you need others kept